README.md

```diff
@@ -262,77 +201,220 @@ builder.toValue();
+`SQONBuilder.in(fieldName: string, value: ArrayFilterValue) => SQONBuilder`
 
-#### SQON.gt(fieldName, value)
+Creates a filter requiring the given field to have one of the given values. The value matches the `ArrayFilterValue` type which representes a string, a number, or an array of strings and numbers.
 
-Greater Than operator. Create a filter requiring the given field to be greater than the given value
+Example: `SQONBuilder.in('name',['Jim','Bob'])`
 
-Example: `SQON.gt('age',21)`
+#### Filter: Greater Than
+`SQONBuilder.gt(fieldName: string, value: number) => SQONBuilder`
 
-#### SQON.lt(fieldName, value)
+Greater Than operator. Create a filter requiring the given field to be greater than the given value.
 
-Lesser Than operator. Create a filter requiring the given field to be lesser than the given value
+Example: `SQONBuilder.gt('age',21)`
 
-Example: `SQON.lt('count', 100)`
+#### Filter: Lesser Than
+`SQONBuilder.lt(fieldName: string, value: number) => SQONBuilder`
 
-### Combinations
+Lesser Than operator. Create a filter requiring the given field to be lesser than the given value.
 
-Combinations can be initiaed from the SQON class or from a SQON instance.
+Example: `SQONBuilder.lt('count', 100)`
 
-If this is called from an instance, the method will accept one SQON or an array, and the instance can be considered grouped with the SQONs provided in the arguments.
-
-If this is called from the class, an array of SQONs is required to be passed.
-
-#### SQON.and(sqon)
+#### Combine: And
+`SQONBuilder.and(sqon: SQON | SQON[]) => SQONBuilder`
 
 All filters in the resulting SQON must be true.
 
-Example: `SQON.and( [someSqon, anotherSqon] )`
+Example: `SQONBuilder.and( [someSqon, anotherSqon] )`
 
-#### SQON.or(sqon)
+#### Combine: Or
+`SQONBuilder.or(sqon: SQON | SQON[]) => SQONBuilder`
 
 At least one filter in the resulting SQON must be true.
 
-Example: `SQON.or( [someSqon, anotherSqon] )`
+Example: `SQONBuilder.or( [someSqon, anotherSqon] )`
 
-#### SQON.not(sqon)
+#### Combine: Not
+`SQONBuilder.not(sqon: SQON | SQON[]) => SQONBuilder`
 
 None of the filters in the resulting SQON can be true.
 
-Example: `SQON.not( [someSqon] )`
+Example: `SQONBuilder.not( [someSqon] )`
 
-### From
+#### From
+`SQONBuilder.from(input: unknown) => SQONBuilder`
 
 Build a new SQON from a string or from a JSON object.
 
 Example with string:
 
 ```ts
-SQON.from(
+SQONBuilder.from(
   '{"op":"and","content":[{"op":"in","content":{"fieldName":"name","value":"Tim"}},{"op":"gt","content":{"fieldName":"age","value":"19"}}]}',
 );
 ```
 
-This will throw an error if the provided value is not a valid SQON or JSON string.
+This differs from the default function because it will accept any input. The use case for this function is when you have a JS object that could potentially be a valid SQON, you can pass this `uncheckedObject` to `SQONBuilder.from(uncheckedObject)` and validation will be performed before the `SQONBuilder` is returned, if possible.
```

Comment:
This says .from behaves differently, but based on the description for SQONBuilder itself, I'm not 100% sure I understand what the difference is here.
```suggestion
This differs from the default function in that it will accept any input. The use case for this function is when you have a JS object that could potentially be a valid SQON, you can pass this `uncheckedObject` to `SQONBuilder.from(uncheckedObject)` and validation will be performed before the `SQONBuilder` is returned, if possible.
```